app/config.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from app.customer import Customer, Car
+from app.shop import Shop
+
+with open("app/config.json", "r") as config_file:
+    data = json.load(config_file)
+
+    FUEL_PRICE = data["FUEL_PRICE"]
+    CUSTOMERS = []
+    SHOPS = []
+
+    Car.fuel_price = FUEL_PRICE
```

Comment:
I'm not sure that this is the best idea. You can pass fuel_price as argument to the method `calculate_ride_cost_to `